packages/amplify-data-construct/src/index.ts

```diff
@@ -14,6 +14,7 @@ import {
   ConflictResolutionStrategy,
   ConflictResolutionStrategyBase,
   CustomConflictResolutionStrategy,
+  CustomSqlDataSourceStrategy,
```

Comment:
I think this will fail until you re-generate the API.md and jsii

Reply:
sigh emoji.

Regenerated.